backend/src/main/kotlin/app/ehrenamtskarte/backend/regions/database/repos/RegionsRepository.kt

```diff
@@ -59,4 +60,15 @@ object RegionsRepository {
             .single().id
         return RegionEntity[regionId]
     }
+
+    fun findRegionByNameAndPrefix(name: String, prefix: String): RegionEntity? =
+        RegionEntity.find { (Regions.name eq name) and (Regions.prefix eq prefix) }.singleOrNull()
+
+    fun findRegionByFreinetId(freinetId: Int): RegionEntity? =
+        (FreinetAgencies innerJoin Regions)
+            .slice(Regions.columns)
+            .select { FreinetAgencies.agencyId eq freinetId }
+            .singleOrNull()?.let {
+                RegionEntity.wrapRow(it)
+            }
```

Comment:
I think it would be good to also filter by project id, to make sure that we do not end up returning a region from a  different project.

Reply:
ah alright and also for both functions. thx for the hint

Reply:
added in last commit